conf/messages

```diff
@@ -602,7 +633,6 @@ email-whitelisting.problem.bullet4=the trial has ended
 email-whitelisting.problem.p3=You’ll need to email HMRC at <strong class="bold">ddcw-info-cgtpd-g@digital.hmrc.gov.uk</strong> to request access to the service.
 email-whitelisting.problem.p4=You should include your name.
 
-
 agent.enter-client-cgt-ref.title=What is your client’s Capital Gains Tax on UK property account number?
```

Comment:
not sure if I'd class this as onboarding - this might be worth taking out into a separate `Agent` section. That's how the code is structured anyway

Reply:
moved it into new AGENT section